src/Core/Domain/Manufacturer/Command/BulkToggleManufacturerStatusCommand.php

```diff
@@ -89,7 +89,7 @@ private function setManufacturerIds(array $manufacturerIds)
     /**
      * Validates that value is of type boolean
      *
-     * @param $value
+     * @param bool $value
```

Comment:
`$value` real type is `mixed`. We _want_ it to be a boolean 😄  but we are not sure which is why we assert it. We expect sometimes people pass something else than a boolean here.

Reply:
The `BulkToggleManufacturerStatusCommand` is used twice in `src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php` and each time, the `expectedStatus` is a boolean.

Reply:
As explained in https://github.com/PrestaShop/PrestaShop/pull/20118/files#r464460113, we cannot rely on our usage of the code. This code has a life of its own outside of PrestaShop main usage because we expect our code to be extended by module and shop developers, which is why it must be adapted to this specific usage.